plugins/query-insights/src/main/java/org/opensearch/plugin/insights/rules/model/Attribute.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights.rules.model;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+import java.util.Locale;
+
+/**
+ * Valid attributes for a search query record
+ */
+public enum Attribute {
+    /**
+     * The search query type
+     */
+    SEARCH_TYPE,
+    /**
+     * The search query source
+     */
+    SOURCE,
+    /**
+     * Total shards queried
+     */
+    TOTAL_SHARDS,
+    /**
+     * The indices involved
+     */
+    INDICES,
+    /**
+     * The per phase level latency map for a search query
+     */
+    PHASE_LATENCY_MAP;
```

Comment:
can you also add node id as an attribute for cordinator node.

Reply:
Added nodeId as an attribute and also incorporate that in top queries results.